### PR TITLE
feat(a11y): Radio a11y keyboard #205

### PR DESCRIPTION
**Pull request by YyumeiZhang**
[中文模板 / Chinese Template](https://github.com/DouyinFE/semi-design/blob/main/.github/PULL_REQUEST_TEMPLATE.zh-CN.md)

- [x] I have read and followed [Pull Request Guidelines](https://github.com/DouyinFE/semi-design/blob/main/CONTRIBUTING-en-US.md#pull-request-guidelines) of the contributing guide.

### What kind of change does this PR introduce? (check at least one)

 - [ ] Bugfix
 - [x] Feature
 - [ ] Code style update
 - [ ] Refactor
 - [ ] Test Case
 - [ ] TypeScript definition update
 - [ ] Document improve
 - [ ] CI/CD improve
 - [ ] Branch sync
 - [ ] Other, please describe:

### PR description

Feat related to #205 

### Changelog
🇨🇳 Chinese
- feat: Radio新增 A11y 键盘适配

---

🇺🇸 English
- feat:  Added A11y keyboard adaptation for Radio

### Checklist
- [x] Test or no need
- [x] Document or no need
- [x] Changelog or no need

### Other
- [ ] Skip Changelog

### CODE REVIEW

File: content/input/radio/index.md [hunk 1]
```diff
@@ -321,6 +325,7 @@ class App extends React.Component {
 | addonId | addon 节点 id，aria-labelledby 指向这个 id，若无设置会随机生成一个 id  **v2.11.0 后提供**                                 | string            |       |
 | addonStyle     | 包裹内容容器的内联样式  **v1.16.0 后提供**                                 | CSSProperties     |       |
 | aria-label      | Radio 的 label                                                            | string           | -  |
+| name         | Radio组件中`input[type="radio"]`的`name`属性                               | string         | -  |
```
[the remark this answers is not in the document]
Comment (shijiatongxue) [reply content/input/radio/index.md:328]: image

File: content/input/radio/index.md [hunk 2]
```diff
@@ -365,16 +370,21 @@ class App extends React.Component {
 
 ## Accessibility
```
Comment (shijiatongxue) [inline content/input/radio/index.md:371]: `## Accessibility` 标题下面写下参考的标准吧。

image

File: content/input/radio/index.md [hunk 3]
```diff
@@ -365,16 +370,21 @@ class App extends React.Component {
...
 - `aria-label`：用于解释 Radio 或 RadioGroup 的作用
 - `aria-labelledby` 默认指向 addon 节点，用于解释 Radio 的内容
 - `aria-describedby` 默认指向 extra 节点，用于补充解释 Radio 的内容
 
+### 键盘和焦点
+
+- RadioGroup 可以被获取焦点，初始焦点设置：
+  - 当 RadioGroup 中没有被选择项时，初始焦点为第一个 Radio 项上；
+  - 当 RadioGroup 中有选中项时，初始焦点为选中的 Radio 项上。
+- 通过 `右箭头` 或 `下箭头` 将焦点移动到下一个 Radio 项上，同时取消先前的 Radio 项的选中状态，并选中当前聚焦的 Radio 项；
```
[the remark this answers is not in the document]
Comment (YyumeiZhang) [reply content/input/radio/index.md:384]: 这里对RadioGroup生效，不在radio group中的radio不能通过上下左右箭头切换

**Review by shijiatongxue — APPROVED** [review]

**Comment by shijiatongxue** [comment]
等待发布 minor 版本后合入 beta。

**Pull request merged**